lib/index.js

```diff
@@ -46,6 +46,11 @@ function downloadFromInfoCallback(stream, info, options) {
     url += '&range=' + options.range;
   }
 
+  doDownload(url,stream);
+}
+
+//For bugfix https://github.com/fent/node-ytdl-core/issues/50; TODO: maybe tidy it up//
```

Comment:
I wouldn't post a link to the issue that sprouted this, I think the comment below explaining why that particular status code is checked is enough.